frontend/readme.md

```diff
@@ -0,0 +1,76 @@
+# FrontEnd For Streamlit
```

Comment:
README.md

Reply:
Run `git rm --cached readme.md` to remove this file.

Reply:
Done!